### PR TITLE
done task 6 and and passed Error handling  in separate file

### PR DESCRIPTION
**Pull request by NikitaSelanko**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -114,4 +116,46 @@ describe('Users tests', () => {
...
+      return request(app)
+        .patch("/api/reviews/1")
+        .send({ inc_votes: "something that's NaN" })
+        .expect(400)
+        .then(({ body: { msg } }) => {
+          expect(msg).toBe("wrong input");
+        });
+    });
+  });
+
```
Comment (HeatherEver) [inline __tests__/app.test.js:178]: please add tests for invalid and non-existent id's for PATCH :)

File: controllers/error-controllers.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+exports.handleIternalErrors = (err, req, res, next) => {
```
Comment (HeatherEver) [inline controllers/error-controllers.js]: small spelling mistake in `Internal` might cause bugs later in life :)

**Review by HeatherEver — APPROVED** [review]
Nice just a couple more tests tot add

File: models/reviews-model.js [hunk 3]
```diff
@@ -6,13 +6,30 @@ exports.selectReviews = () => {
...
       if (review) {
+       
         return review;
       }
       return Promise.reject({ status: 404, msg: "ID NOT FOUND" });
     });
 };
+
+exports.updateVotes = (review_id, votes) => {
+  if(!votes) {
```
Comment (scar1377) [inline models/reviews-model.js:22]: can do if(!votes.inc_votes). that will cover a body with missing key like {newVotes:1}

**Review by scar1377 — APPROVED** [review]
Good to merge

**Pull request merged**